addons/crm/views/crm_lead_views.xml

```diff
@@ -864,7 +864,9 @@
             <field name="binding_view_types">list</field>
             <field name="state">code</field>
             <field name="code">
-if record:
+if record.type == 'lead':
```

Comment:
What happens if you select 20 records, both lead and opportunities ^^ ?

Reply:
hmm adding `or record.length > 1` to the elif will keep the same behavior as before the fix, do we want that?

Reply:
(interjecting :chicken: since I'm passing by)
@Ephoniz You could just do something like `if not 'opportunity' in record.mapped('type'):`

We want a lost reason for opportunities, if there happens to be leads with that opportunity they can still get a lost reason. Not asking for lost reasons on leads is a matter of convenience.

Reply:
@thiryr Yes, that's more clean I think, thanks 😄